app/commands/site_updates/process_new_exercise_update.rb

```diff
@@ -0,0 +1,40 @@
+module SiteUpdates
+  class ProcessNewExerciseUpdate
+    include Mandate
+
+    initialize_with :exercise
+
+    def call
+      if exercise.wip?
```

Comment:
What happens in the unlikely case that a new exercise is created with the `deprecated` status?